Task1/aga/IsPrime/IsPrime.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
Exactly. This means all the files in Task1/aga/IsPrime/IsPrime/obj/Debug should be removed from version control and added to .gitignore file.